mantle/kola/tests/misc/boot-mirror.go

```diff
@@ -25,6 +26,7 @@ import (
 	"github.com/coreos/mantle/platform/conf"
 	"github.com/coreos/mantle/platform/machine/unprivqemu"
 	"github.com/coreos/mantle/system"
+	ut "github.com/coreos/mantle/util"
```

Comment:
Is there a problem with just leaving this without the `ut` and just using `util.Retry`?

Reply:
We're already using the `util` package [here](https://github.com/coreos/coreos-assembler/pull/2628/files#diff-71e0d17842ab236a51a8b1d495515b44ac49f04f3bf166bae6cf555e66a5b48fR24)

Reply:
k